zerver/migrations/0453_alter_scheduledmessagenotificationemail_trigger.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 4.2.1 on 2023-06-03 10:51
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("zerver", "0452_realmauditlog_extra_data_json"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="scheduledmessagenotificationemail",
+            name="trigger",
+            field=models.TextField(
+                choices=[
+                    ("private_message", "Private message"),
+                    ("mentioned", "Mention"),
+                    ("stream_wildcard_mentioned", "Stream wildcard mention"),
+                    ("stream_email_notify", "Stream notifications enabled"),
+                ]
+            ),
+        ),
+    ]
```

Comment:
So this migration is a quirk of Django -- this migration file doesn't actually change the database at all; it just exists to tell the Django migration code that the new choice is a possible valid value in case another migration interacts with it. Rather than create a new migration that doesn't actually change the database, we should edit the last migration touching this field to just have the extra choice included.

Reply:
I think we need to change the value stored in the DB as "wildcard_mentioned" to "stream_wildcard_mentioned" ?

Reply:
Yeah, changing existing values in the database sounds correct -- but I believe this migration file just changes the set of values that Django's Python implementation thinks are valid values, without changing existing values in the database at all.

Reply:
We'll need to do two things here:
1. Edit the possible choices to what we want. Edit 0331_scheduledmessagenotificationemail.py as well. Reasoning: https://chat.zulip.org/#narrow/dm/16839,22341-dm/near/1592884
2. To update the existing values in the DB, create a custom migration. See https://github.com/zulip/zulip/pull/24354/commits/0cfa9c96f58841f531987a9422f118ea6fed594d for an example.

Reply:
@timabbott does that sound right?

Reply:
In the first commit:
1.[ Edit 0031_scheduledmessagenotificationemail.py](https://github.com/zulip/zulip/pull/25828/commits/b4aa15502d467199e2ee706def223dc21b988abb#diff-932e136b53606abc4de3a5f5fded9fb351850927c044dc24812c98eab2773843)
2. [Update the existing values in DB](https://github.com/zulip/zulip/pull/25828/commits/b4aa15502d467199e2ee706def223dc21b988abb#diff-9f450075da1f52de9653a804b2dfb3d74ae674f68df480c98e897b1d7d036e6a)

Reply:
Yeah that's correct.